site/src/main/scala/org/typelevel/sbt/TypelevelSitePlugin.scala

```diff
@@ -256,7 +256,7 @@ object TypelevelSitePlugin extends AutoPlugin {
 
       val (_, cancel) = Tasks.buildPreviewServer.value.allocated.unsafeRunSync()
 
-      logger.info(s"Preview server started on port ${laikaPreviewConfig.value.port}.")
+      logger.info(s"Preview server started on port http://localhost:${laikaPreviewConfig.value.port}.")
```

Comment:
We're currently discarding the server in the `_`, we can actually get the address/port directly out of that instead of hard-coding.